src/build_log.cc

```diff
@@ -12,6 +12,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// On AIX, inttypes.h gets indirectly included by build_log.h.
+// It's easiest just to ask for the printf format macros right away.
+#ifndef _WIN32
```

Comment:
Should this check for _WIN32 or _MSC_VER? Would MinGW want the __STDC_FORMAT_MACROS define?